examples/with-eip-1193-provider/src/lib/utils.ts

```diff
@@ -94,3 +94,22 @@ export const registerPassKey = async (
 
   return { challenge: base64UrlEncode(challenge), attestation };
 };
+
+export const estimateFees = async () => {
```

Comment:
we might want to pass the chain ID here so that we can estimate on the correct network (it's not a _huge_ deal though given mainnet gas prices are typically at least testnet's gas prices)